### DISCUSSION TITLE
Shutting down the Discord Server

### DISCUSSION THREAD

**Discussion by Kludex** [pydantic/pydantic#8624] [General]
It looks like we didn't have much engagement on our discord server. It may make sense for us to shutdown the server.

A lot of questions are lost on the server, and it may make more sense for us to recommend people to just create questions here (in the discussions).

Are there strong opinions against this?

**Comment by Kludex** [comment]
I've shut down the server.

**Reply by ecryth** [reply]
Well that explains why the invite was invalid when I went to join just now, that's some incredible timing haha. I was hoping to use it to quickly discuss some ideas for a contribution I'm looking to make to Pydantic, but from the sound of it it wasn't active enough to be useful anyway.

If the Discord didn't work out, what's the best channel currently to get some quick feedback on a potential PR? Would you recommend just opening a draft PR with my proof of concept and some accompanying questions?

**Reply by samuelcolvin** [reply]
Yes, best to create an issue or draft PR.

**Reply by SDAravind** [reply]
It would be great to have some social platform where users can interact and ask questions.

**Reply by laisbsc** [reply]
> It would be great to have some social platform where users can interact and ask questions.

@SDAravind join Slack: https://logfire.pydantic.dev/docs/join-slack/

**Comment by mg3146** [comment]
Doh - I've kept wanting to use the pydantic discord to ask simple questions.  shame it was shut down

**Reply by laisbsc** [reply]
> Doh - I've kept wanting to use the pydantic discord to ask simple questions.  shame it was shut down

@mg3146 If you're looking for a place to ask questions, join the Slack server: https://logfire.pydantic.dev/docs/join-slack/
The whole engineering team is there.
